lib/vmpooler/api/v1.rb

```diff
@@ -240,13 +270,25 @@ def validate_date_str(date_str)
           result[task][:duration][:total] += daily[task][:duration][:total]
 
           if result[task][:count][:total] > 0
```

Comment:
This block should be outside the loop so that the total duration average is only calculated once, not altered each day.